aries-site/src/pages/components/button.mdx

```diff
@@ -54,6 +54,18 @@ Using the design system Button Component will ensure your button looks like a bu
 - Avoid contractions or colloquialisms. Keep the language approachable but not personal, for examples, see [point of view](https://design-system.hpe.design/foundation/voice-and-tone?q=tone#point-of-view).
 - Ensure your button label matches the route destination.
 
+#### Guidance on when to use 'add', 'create' and 'new'.
+
+**Add**: ‘Add’ should be used where someone can add something that is already existing. It’s recommended to include the name of the thing that user can create. Combine add with the object where appropriate, Add+thing to create. 
+Examples: Add user, Add file, or Add member. 
+
+**New**: Use ‘New’ for creating something new, a new object Include the name of the thing that user can create.
+Use ‘New’ to initiate the action. 
+Examples: New account, New file or New tab.
+
+**Create**: ‘Create’ should be used where the user wants to create something that needs to be produced from the scratch, from nothing. Create requires users to include details as part of the create process. Create can be used for submit button of a form.  
+Example: Create a design, Create a project, or Create volume. 
```

Comment:
```suggestion
Examples: Create a design, Create a project, or Create volume. 
```

Reply:
Agreed, 'a' should be omitted.
Should I delete and resubmit?

Reply:
Agreed we can remove the 'a'.